file/filename.cc

```diff
@@ -24,6 +24,18 @@ namespace rocksdb {
 static const std::string kRocksDbTFileExt = "sst";
 static const std::string kLevelDbTFileExt = "ldb";
 static const std::string kRocksDBBlobFileExt = "blob";
+static const std::string kEncryptionSkipName  = "CURRENT";
+
+bool ShouldSkipEncryption(const std::string& fname) {
+  size_t check_length = kEncryptionSkipName.length();
```

Comment:
check equal against "CURRENT" (can be hardcoded) and check suffix against `kUnencryptedTempFileNameSuffix`